request_a_govuk_domain/request/admin/model_admins.py

```diff
@@ -579,9 +613,14 @@ def get_field_names(self):
             "date_submitted",
             "registrar_org",
             "domain_name",
```

Comment:
Maybe you could rename those to be more human readable? "Domain name", "Registrar", "Date submitted", etc.